hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestObserverWithRouter.java

```diff
@@ -122,7 +123,9 @@ public void startUpCluster(int numberOfObserver, Configuration confOverrides) th
 
     cluster.waitActiveNamespaces();
     routerContext  = cluster.getRandomRouter();
-    fileSystem = routerContext.getFileSystemWithObserverReadsEnabled();
+    Configuration confToEnableObserverRead = new Configuration();
+    confToEnableObserverRead.setBoolean(HdfsClientConfigKeys.DFS_RBF_OBSERVER_READ_ENABLE, true);
+    fileSystem = routerContext.getFileSystem(confToEnableObserverRead);
```

Comment:
We are losing coverage on `getFileSystemWithObserverReadsEnabled` with this change; we should likely be testing both as they are both valid use-cases whether you want to msync or not.